api/envoy/api/v2/core/protocol.proto

```diff
@@ -79,6 +79,12 @@ message Http2ProtocolOptions {
 
   // Allows proxying Websocket and other upgrades over H2 connect.
   bool allow_connect = 5;
+
+  // [#not-implemented-hide:] Hiding until envoy has full metadata support.
+  // Still under implementation. DO NOT USE.
+  //
+  // Allows metadata. See source/docs/h2_metadata.md for more information.
```

Comment:
These docs get published on the envoy proxy site, so I think we'll want real web links so folks can click through

so I think for you something like
See \[metadata docs](https://github.com/envoyproxy/envoy/blob/master/source/docs/h2_metadata.md) for more information.

Reply:
I see! Fixed.